R/get_data.R

```diff
@@ -105,6 +105,21 @@
 #' )
 #' }
 #'
+#' ## Pass an argument through to a read_* function
+#'
+#' try(
+#'   bcdc_get_data(record = "a2a2130b-e853-49e8-9b30-1d0c735aa3d9",
+#'                 resource = "0b9e7d31-91ff-4146-a473-106a3b301964")
+#' )
+#'
+#' ## we can control some properties of the list object returned by
+#' ## jsonlite::read_json by setting simplifyVector = FALSE or
```

Comment:
`FALSE` is the default for these, so should this be `TRUE`?

Reply:
🤦‍♀️ it was correct in the code, but not in the text. Great catch. Fixed.